logstash-core/lib/logstash/util/wrapped_acked_queue.rb

```diff
@@ -332,10 +332,6 @@ def flush_signal_received?
 
       private
 
-      def iterating?
-        @iterating
-      end
```

Comment:
@jakelandis hmm theoretically yes since JRuby doesn't inline this stuff and it appears to translate into 2 function calls. If it were just Java the answer is no, JVM inlines pure accessors every time, but on JRuby it seems to add overhead.